src/index.js

```diff
@@ -3,30 +3,40 @@ var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 var API = 'https://rickandmortyapi.com/api/character/';
 var xhttp = new XMLHttpRequest();
 
-function fetchData(url_api, callback) {
-  xhttp.onreadystatechange = function (event) {
-    if (xhttp.readyState === '4') {
-      if (xhttp.status == 200)
-        callback(null, xhttp.responseText);
-      else return callback(url_api);
+function fetchData(url_api) {
+  return new Promise((resolve, reject) => {
+    xhttp.onreadystatechange = event => {
+      if (xhttp.readyState === 4) {
+        if (xhttp.status == 200){
```

Comment:
Dejemos un espacio entre el valor a comparar y las primeras llaves { }
```suggestion
        if (xhttp.status == 200) {
```

Reply:
Gracias :-)